haystack/json-schemas/haystack-pipeline-1.10.0rc0.schema.json

```diff
@@ -3044,10 +3044,10 @@
               "title": "Use Auth Token",
               "anyOf": [
                 {
-                  "type": "boolean"
```

Comment:
I am not sure why the order changed here but I hope that the next PR doesn't change it back again.